mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -1492,15 +1507,13 @@ impl TakerSwap {
             self.p2p_privkey,
         );
 
-        let wait_taker_payment =
-            wait_for_taker_payment_conf_until(self.r().data.started_at, self.r().data.lock_duration);
```

Comment:
As I can see, `wait_for_taker_payment_conf_util` returned
```rust
swap_started_at + (locktime * 4) / 5
```
But `self.r().data.taker_payment_lock` is
https://github.com/KomodoPlatform/atomicDEX-API/blob/6e4de5d217c654057bee49b283737b7ad4499734/mm2src/mm2_main/src/lp_swap/taker_swap.rs#L885

Is this expected?

Reply:
Yes, normally if the maker does not spend the taker payment until `swap_started_at + (locktime * 4) / 5` the taker was going to the `refund_taker_payment ` method and just sleeping until the locktime. After a discussion with @artemii235, we decided to change it such that the taker instead waits for the taker payment spend until `swap_started_at + locktime`, then directly starts the refund process afterwards.